apigetpost.py

```diff
@@ -86,17 +92,18 @@ def api_get_post(post_url):
         post_data.owner_name = ""
         post_data.owner_url = ""
         post_data.owner_rep = 1
+
     post_data.site = site
     post_data.body = item['body']
     post_data.body_markdown = item['body_markdown']
     post_data.score = item['score']
     post_data.up_vote_count = item['up_vote_count']
     post_data.down_vote_count = item['down_vote_count']
     post_data.creation_date = item['creation_date']
-    try:
-        post_data.last_edit_date = item['last_edit_date']
-    except KeyError:
-        post_data.last_edit_date = post_data.creation_date  # Key not present = not edited
+    
+    post_data.last_edit_date = item.get('last_edit_date', post_data.creation_date)  # Use .get() for safer access
```

Comment:
This line is a direct equivalent to the `try`/`except` block which it is replacing. There's significant discussion in the Python world as to which is "better", for various values of "better".  Personally, I prefer the `.get()`, but not enough to have previously pushed the change, but I wouldn't object to it being changed.

However, I would retain the functional comment that's in the old code, rather than have the comment that's been added here, which is just a comment on how to write code. In other words, the old comment is a reasonable code comment. The added comment is something that would be more appropriate in a commit comment instead of a code comment.